import_flow_events.py

```diff
@@ -32,17 +32,22 @@
 EVENTS_PREFIX = "fxa-flow/data/"
 EVENTS_FILE_URL = "s3://" + EVENTS_BUCKET + "/" + EVENTS_PREFIX + "flow-{day}.csv"
 
-# We import each into is own table and maintain a UNION ALL view over them.
+# There are three tables:
+#   * flow_csv      - raw data from the CSV file
+#   * flow_metadata - metadata for each flow
+#   * flow_events   - individual flow events
 
-Q_CREATE_TABLE = """
-    CREATE TABLE IF NOT EXISTS flow_events (
-      timestamp BIGINT NOT NULL sortkey, 
-      flowid VARCHAR(64) distkey,
+Q_DROP_CSV_TABLE = "DROP TABLE IF EXISTS flow_csv;"
+
+Q_CREATE_CSV_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_csv (
+      timestamp BIGINT NOT NULL SORTKEY,
       type VARCHAR(30) NOT NULL,
-      flowTime INTEGER,
-      userAgentBrowser VARCHAR(40),
-      userAgentVersion VARCHAR(40),
-      userAgentOS VARCHAR(40),
+      flowId VARCHAR(64) NOT NULL DISTKEY,
```

Comment:
I've set the order of this table to directly match the order of the CSV file. Figured that may save confusion down the line (or maybe that's just me, I got a bit confused before I did this earlier).